app/config.go

```diff
@@ -13,9 +14,18 @@ var ConfigFilename = "config.json"
 
 // SetupConfig loads the configuration in the given struct. In case of error, prints help and exit application.
 func SetupConfig(config interface{}) {
-	files := make(uconfig.Files)
+	var files uconfig.Files
 	if fileExists(ConfigFilename) {
-		files[ConfigFilename] = json.Unmarshal
+		files =
```

Comment:
What is the purpose of this?

Reply:
This line break shouldn't exist... but regarding the refactor, if you check the type for `uconfig.Files` you get this:

`type Files = file.Files`

and if you check the file.Files type

`// Files represents a set of file paths and the appropriate
// unmarshal function for the given file.
type Files []struct {
	Path      string
	Unmarshal Unmarshal
}`

In the previous version this was a map but now it's a slice of struct. So I'm just creating one with the config file, if it exists. Just like the previous version did.